app/v3/retrievePeriodSummary/RetrievePeriodSummaryConnector.scala

```diff
@@ -50,8 +47,13 @@ class RetrievePeriodSummaryConnector @Inject() (val http: HttpClient, val appCon
 
     request match {
       case _: Def1_RetrievePeriodSummaryRequestData =>
-        val downstreamUri = DesUri[Def1_RetrievePeriodSummaryResponse](
-          s"income-tax/nino/$nino/self-employments/$businessId/periodic-summary-detail?from=$fromDate&to=$toDate")
+        val downstreamUri =
+          if (featureSwitches.isDesIf_MigrationEnabled)
+            IfsUri[Def1_RetrievePeriodSummaryResponse](
+              s"income-tax/nino/$nino/self-employments/$businessId/periodic-summary-detail?from=$fromDate&to=$toDate")
```

Comment:
Because the path is identical in both cases, it could be put in a single val, then reused